valohai_yaml/schema/task.yaml

```diff
@@ -0,0 +1,28 @@
+type: object
+required:
+  - name
+  - step
+  - parameters
+additionalProperties: false
+properties:
```

Comment:
Should we have (optional) task configuration being part of this or have you planned it to come later?

For every Task we have `stop_expression` text and `maximum_queued_executions` integer
For random search we also have `execution_count` integer
and for Bayesian we have 
- `execution_count` integer
- `execution_batch_size` integer
- `optimization_target_metric` text
- `optimization_target_value` float
- `engine` text

Reply:
Should not this all be in step template? I thought to make a task template more related on step and to add only task-specific property into template.

Reply:
These should be task specific configurations nothing to do with the step.